2024-Wrapped.ipynb

```diff
@@ -0,0 +1,856 @@
+       "2023-09-02    Talk  02:16:27     0         1       1       1  \n",
+       "2023-08-21    Talk  01:47:35     1         0       0       1  \n",
+       "2023-08-10    Talk  01:55:33     0         1       1       0  \n",
+       "2023-07-20    Talk  01:59:23     1         0       1       1  \n",
+       "2023-06-30    Talk  02:00:27     1         0       1       1  \n",
+       "2023-06-15    Talk  02:53:56     1         1       1       1  \n",
+       "2023-05-19    Talk  03:34:07     1         1       1       1  \n",
+       "2023-04-21    Talk  02:31:03     1         1       1       1  \n",
+       "2023-03-28    Talk  02:13:14     1         0       1       1  \n",
+       "2023-03-10    Talk  01:32:17     1         0       1       1  \n",
+       "2023-02-18    Talk  01:36:49     1         0       1       1  \n",
+       "2023-01-28    Talk  01:33:42     1         1       1       1  \n",
+       "2023-01-06    Talk  01:55:32     1         1       1       1  "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "talk = podcasts[podcasts.Type == \"Talk\"]\n",
+    "talk.drop(columns=['pfleidi', 'marc'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Sendungen pro Host:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ingo:     17\n",
+      "l33tname: 13\n",
+      "madmas:   19\n",
+      "makefu:   16\n"
+     ]
+    }
+   ],
+   "source": [
+    "ingo_count = talk[\"ingo\"].sum()\n",
+    "l33tname_count = talk[\"l33tname\"].sum()\n",
+    "madmas_count = talk[\"madmas\"].sum()\n",
+    "makefu_count = talk[\"makefu\"].sum()\n",
+    "\n",
+    "\n",
+    "print(f\"ingo:     {ingo_count}\")\n",
+    "print(f\"l33tname: {l33tname_count}\")\n",
+    "print(f\"madmas:   {madmas_count}\")\n",
+    "print(f\"makefu:   {makefu_count}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In Prozen sind das:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ingo:      85%\n",
+      "makefu:    80%\n",
+      "l33tname:  65%\n",
+      "madmas:    95%\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"ingo:      {ingo_count/talk_count:.0%}\")\n",
+    "print(f\"makefu:    {makefu_count/talk_count:.0%}\")\n",
+    "print(f\"l33tname:  {l33tname_count/talk_count:.0%}\")\n",
+    "print(f\"madmas:    {madmas_count/talk_count:.0%}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Damit ist madmas MVP der gerade mal 1 Sendung verpasst hat."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Duration"
```

Comment:
a lot of durations are wrong with 0:00:00 which breaks a lot in this section

Reply:
updated the csv

Reply:
updated the csv